docs/source/run/parameters.rst

```diff
@@ -146,6 +146,17 @@ Modeling ion motion is not yet supported by the explicit solver
     The small dst is quicker for simulations with :math:`\geq 511` transverse grid points.
     The default is set accordingly.
 
+* ``fields.extended_solve`` (`bool`) optional (default `0`)
+    Extends the area of the FFT Poisson solver to the ghost cells. This can reduce artefacts
```

Comment:
Little typo: In English with an `i`:
```suggestion
    Extends the area of the FFT Poisson solver to the ghost cells. This can reduce artifacts
```

Reply:
Fixed